### DISCUSSION TITLE
Congratulations!!!

### DISCUSSION THREAD

**Discussion by ccarlo88** [TechnitiumSoftware/DnsServer#248] [General]
Hello!

Just passing by to say congratulations for the work. Your project looks really nice!
I will try it in my Ubuntu private server in some days but I can say I am already happy about not having to deal with some strange text file to configure it... Also using C# it is very likely that it will be easier to maintain. Great job!

**Comment by ShreyasZare** [comment]
Thanks for the compliments. Do give your feedback.

**Reply by animehunter123** [reply]
Hello, I wanted to also say thank you for your wonderful application. Please continue to keep the DNS Server open source and free, as it helped me very much with my lab testing and learning in general.

Kind regards

**Reply by ShreyasZare** [reply]
Thanks for the compliments!
